### PR TITLE
feat: Create a database schema diagram.

### PR DESCRIPTION
**Pull request by simplegoose**
Hi :wave: 

We have completed the requirements for the clinic database diagram. We have created a file named schema_based_on_diagram.sql which implements the database from the diagram. We have also included the necessary tables for many-to-many relationships, even if they were not explicitly shown in the diagram. Additionally, we have added the FK indexes for the tables.

Schema Diagram

Thank you for your time.
Best regards,
@rashedarman and @simplegoose

### CODE REVIEW

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission :red_circle:

invalid-submission

## Hey @simplegoose, @rashedarman :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements but I cannot proceed review of your pull request because there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

# Mandatory Changes ♻️
- [ ] Kindly add the following sections with meaningful information in them in the readme file to make your readme descriptive and professional 😊

* Key Features
* Future Features
* Acknowledgement

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  PRIMARY KEY (medical_history_id, treatment_id)
+);
+
+CREATE TABLE invoices (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  total_amount NUMERIC(10, 2),
+  generated_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_history_id INT REFERENCES medical_histories(id)
+);
```
Comment (GedeonTS) [inline schema_based_on_diagram.sql:42]: - Kindly make sure to add Foreign Key Indexes, according to [this requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.). You can implement that like this: `CREATE INDEX ON medical_histories (patient_id);` and do the same for all Foreign Keys

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @rashedarman @simplegoose , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- Good PR Description :+1: 
- Good Gitflow Implementation :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**